com.dropbox.Client.json

```diff
@@ -96,9 +96,9 @@
                     "only-arches": [
                         "x86_64"
                     ],
-                    "url": "https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86_64-74.4.115.tar.gz",
```

Comment:
Interesting: https://www.dropbox.com/download?plat=lnx.x86_64 currently redirects to Location: https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86_64-75.4.141.tar.gz not .142 and that's what the automated updater saw on #35. However, I just found https://www.dropboxforum.com/t5/Desktop-client-builds/bd-p/101003016 which does indeed list .142 as the latest.